ui/raidboss/raidboss_config.ts

```diff
@@ -1195,7 +1195,7 @@ class RaidbossConfigurator {
     const evalTrigger = (trig: LooseTrigger, key: OutputKey, data: RaidbossData) => {
       try {
         const result = this.valueOrFunction(trig[key], data, kFakeMatches, fakeOutputProxy);
-        if (!result)
+        if (result === null || result === undefined)
```

Comment:
I should probably get rid of all this code.  This was all here to give an idea of trigger output prior to output strings existing, and it doesn't show up if there are output strings.  Not for this PR obviously, but foof this is old and super hacky.